### ISSUE TITLE
onLayoutChange

### ISSUE THREAD

**Issue by plandem** [STRML/react-grid-layout#155]
stopped to call on drag/resize.

Although with previous beta version everything works.
```
	constructor(props) {
		super(props);

		this.renderGrid.bind(this);
		this.drawMatrix.bind(this);

		this.state = {
			items: [],
			layout:[]
		};
	}

	onLayoutChange(layout) {
		console.log('onLayoutChange=', layout);
		this.setState({ layout: layout });
}
	renderGrid() {
		console.log('renderGrid=', this);
		//console.log(this.state.layout);
		var removeStyle = {
			position: 'absolute',
			right: '2px',
			top: 0,
			cursor: 'pointer'
		};

		const self = this;
		return this.state.items.map(function(item, i) {
			let id = item.id;
			return (
				<div key={i} _grid={item}>
					<span className="text">{id}</span>
					<span className="remove" style={removeStyle} onClick={self.onRemoveItem.bind(self, id)}>x</span>
				</div>
			);
		});
	}

	onRemoveItem(i) {
		console.log('removing', i);
		//this.setState({items: _.reject(this.state.items, {i: i})});
	}

	onAddItem() {
		const items = this.state.items;

		console.log('adding i=' +  items.length);
		const max_y = items.reduce(function(max_y, item) { return Math.max(max_y, item.y + item.h) }, 0);
		const item = {
			id: 'n' + items.length,
			x: 0,
			y: max_y,
			w: 4,
			h: 1,
			isDraggable: true,
			isResizable: true
		};

		this.setState({ items: items.concat(item) });
	}

render() {
		return (
			<div>
				<button onClick={this.onAddItem.bind(this)}>add dash</button>
				<div>
					<div style={{ width: 600, float: 'left' }}>
						<Grid className="layout"
							  			layout = {this.state.layout}
										 cols={4}
										 rowHeight={100}
										 autoSize={true}
										 minH = {1}
										 minW = {1}
										 verticalCompact = {true}
										 isDraggable = {true}
										 isResizable = {true}
										 useCSSTransforms = {true}
										 margin = {[10, 10]}
										 onLayoutChange = {this.onLayoutChange.bind(this)}
						>{this.renderGrid()}</Grid>
					</div>
					<div id="matrix">{this.drawMatrix()}</div>
				</div>
			</div>
		)
	};
```

**Comment by menelike** [comment]
this should be fixed with https://github.com/STRML/react-grid-layout/commit/27e77a6c3f62bde6312ebc9942083a608f9b551f. This has not been published to npm yet.

**Comment by plandem** [comment]
i hope it will happen soon, because now it's impossible to use it :(

**Comment by STRML** [comment]
It will - pushing forward some other bugfixes to make a better 0.10.2
release.

**Comment by STRML** [comment]
Fixed in `0.10.2`.
